src/get-data.ts

```diff
@@ -7,12 +7,6 @@ import { PRData } from './types';
 const getData = async (): Promise<PRData> => {
   const authToken = getInput('coverageToken');
 
-  if (!authToken) {
-    warning(
```

Comment:
when I click on the hyperlink to this file I can see the warning, but it is not imported if you go to the most recent version of the branch and I haven't made any commits here since you made this comment . your comment is referencing an old version of the file. weird